### ISSUE TITLE
server could fill in url into template, no need for hard coding

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#44]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/44) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

We should make a review how much server requests are needed to get an image.

Currently the get_capabilities.pt template needs lines of having an xlink definition to the servers Url. 

```
#!xml

<OnlineResource xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="h
ttp://www.your.server"/>

 <OnlineResource xmlns:xlink="http://www.w3.org/1999/
xlink" xlink:href="http://www.your.server/?"/>
```
Because the UI does read this for building its base_url when we request a map.
by wms_control.py (getmap)

```
#!python

base_url = self.getOperationByName('GetMap').methods[method]['url']
```

getCapabilities does it by reading the user input

```
#!python

base_url = str(self.cbURL.currentText())
```

In wms_control.get_map we have the Url already on self.url
on some other places as self.wms.url

I would prefer if we could abstain to add the Url to the server configuration template files.

**Comment by ReimarBauer** [comment]
**Original comment by Marc R. (Bitbucket: [wxmetvis](https://bitbucket.org/wxmetvis), ).**

----------------------------------------

Just make sure that everything stays compatible with the WMS standard -- server and UI are also used with other WMS compatible clients and servers.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Will try to get the server to enter that url he uses to that file by parameter.

The benefit is that we can have these files static and no need to alter them.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

parameter server_url introduced to fill in servers url into the template, fixes #44
